prompt-management/features/essential/analytics.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: "Analytics"
+description: "Monitor prompt performance and usage with comprehensive analytics"
+---
+
+LangWatch provides analytics to help you understand how your prompts are performing in production.
```

Comment:
this whole page requires waay more work to be actually exciting, a bunch of pictures with a bunch of different custom graphs cutting and dicing the data in different graphs formats, plotting metadata, showing cost per model, cost per prompt, etc etc, would make this way more valuable and interesting

not to mention graphs on custom evaluation scores and user events, as those are closer to real quality and ROI measurements

and maybe all I said is for analytics overall not just prompts, but we could do all those "grouped by" (custom graphs) or "filtered by" prompt ids